src/common/components/Input/index.tsx

```diff
@@ -0,0 +1,84 @@
+'use client';
+
+import Image from 'next/image';
+import { forwardRef, useState } from 'react';
+
+import { ICON_URL } from '@/common/constants';
+import { cn } from '@/common/utils';
+
+import { InputProps } from './types';
+import { Text } from '../Text';
+
+export const Input = forwardRef<HTMLInputElement, InputProps>(
+  (
+    {
+      label,
+      name,
+      errorMessage,
+      width,
+      height,
+      onFocus,
+      onBlur,
+      suffix,
+      ...props
+    }: InputProps,
+    ref
+  ) => {
+    const [isFocus, setIsFocus] = useState(false);
+
+    return (
+      <div className="relative flex w-full flex-col gap-6px">
```

Comment:
```suggestion
      <div className={cn("relative flex w-full flex-col gap-6px", className)}>
```

`<Input className='mt-12px'/>`
이런식으로 사용하기 위해서 className도 prop으로 받으면 어떨까요?

Reply:
tailwind merge덕분에 너비 값 넣게되면 오버라이딩 되겠네요. 확실히 편하긴 하네요,,!

Reply:
넵 유용하게 쓰고있네요~ [79f6b43](https://github.com/depromeet/toks-web/pull/310/commits/79f6b431790d208d0ed887bf01bbdc601ef33063) 반영했습니다!